pkg/util/bandwidth/linux.go

```diff
@@ -129,6 +149,10 @@ func asciiCIDR(cidr string) (string, error) {
 	return fmt.Sprintf("%s/%d", ip.String(), size), nil
 }
 
+func isIPv6CIDRString(cidr string) bool {
```

Comment:
I have used this function before but it doesn't function as expected. For cidr like `::ffff:1.2.3.4/128`, `net.IsIPv6CIDRString`  will treat it as an IPv4 address, while `tc` still use `ipv6` protocol and `ip6` filter.

This is the test that covers such case.
https://github.com/chendotjs/kubernetes/blob/3b30b38d05a9a7434d089959fd0475cf3a72f0d4/pkg/util/bandwidth/linux_test.go#L996

Reply:
The PR is updated as your suggestion, please take a look if you get time @johscheuer 
I'm afraid it would be to hard to read or maintain when mixing IPv4 and IPv6 together. The  judgement of whether a cidr is IPv6 is always necessary throughout the combined generic function.

Reply:
Can we update kubernetes/utils to handle this scenario?

Reply:
IMO, `tc` matcher is a special scenario.

Reply:
Shouldn't we file this against the golibrary itself? aren't we saying that`net.ParseCIDR(...)` is buggy?

Reply:
It's plausible to claim that `::ffff:1.2.3.4/128` is an IPv4 address, and it's plausible to claim that it's an IPv6 address. It depends why you're asking...